packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/list/index.js

```diff
@@ -0,0 +1,22 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'ovhManagerOctaviaLoadBalancerDashboardPoolsLazyLoading';
```

Comment:
Careful, you have the same name in pools/index.js and it may have unexpected effects